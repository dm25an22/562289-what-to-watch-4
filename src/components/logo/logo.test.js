import React from "react";
import renderer from "react-test-renderer";
import Logo from "./logo.jsx";
import {Router} from "react-router-dom";
import {history} from "../../history";


it(`render Logo component`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <Logo />
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`render Logo component with class "logo__link--light"`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <Logo
          className={`logo__link--light`}
        />
      </Router>

  ).toJSON();

  expect(tree).toMatchSnapshot();
});
