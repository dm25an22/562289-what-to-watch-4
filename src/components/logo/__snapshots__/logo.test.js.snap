// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Logo component 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link undefined"
    href="/"
    onClick={[Function]}
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;

exports[`render Logo component with class "logo__link--light" 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link logo__link--light"
    href="/"
    onClick={[Function]}
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;
